frontend/mobilityai_react/App/Screens/PatientData.js

```diff
@@ -4,15 +4,18 @@ import { Text, View, ScrollView, TouchableHighlight, StyleSheet, ART } from 'rea
 const { Group, Shape, Surface } = ART;
 
 import * as d3 from 'd3'
-import * as scale from 'd3-scale'
-
+// import * as scale from 'd3-scale'
 
 import GetDate from './GetDate.js';
 import Circle from './PatientCircles';
 import BarGraph from './PatientBarGraphs';
 
+import { GetPatientActivities } from '../Lib/Api';
+
+//TODO: Remove temporary data once we get proper data from the server
```

Comment:
Can we remove the temporary data now?

Reply:
Not yet, we'll do it once the server is updated with the new API